Signal/src/call/CallService.swift

```diff
@@ -1026,9 +1038,26 @@ protocol CallServiceObserver: class {
         // It's only safe to access the class properties on the signaling queue, so
         // we dispatch there...
         CallService.signalingQueue.async {
-            Logger.info("\(self.TAG) \(#function): \(self.shouldHaveLocalVideoTrack())")
+            guard let call = self.call else {
+                return
```

Comment:
It's expected for either `call` or `peerConnectionClient` to be nil at this point - for example, this method is called when call is set to nil.